tests/services/test_strategy_marketplace_integration.py

```
@@ -34,6 +36,7 @@
 from app.services.strategy_marketplace_service import (  # noqa: E402
     StrategyMarketplaceService,
 )
+import app.services.strategy_submission_service as strategy_submission_module  # noqa: E402
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)